### PR TITLE
Add DuplicatedEmail Message Validation

### PR DESCRIPTION
**Pull request by Lordapa**

### CODE REVIEW

File: src/DonateTo.IdentityServer/Controllers/AccountController.cs [hunk 1]
```diff
@@ -203,6 +200,28 @@ private async Task<LoginViewModel> BuildLoginViewModelAsync(LoginInputModel mode
...
+            {
+                if (error.Code == duplicatedEmailCode || error.Code == duplicatedUserNameCode)
+                {
+                    if (error.Code != duplicatedUserNameCode)
+                    {
+                        error.Description = "Email registered - please enter another email or sign in.";
+                        ModelState.TryAddModelError(error.Code, error.Description);
+                    }
+                }
+                else
```
Comment (nicolasgranata) [inline src/DonateTo.IdentityServer/Controllers/AccountController.cs:218]: maybe we can move the line 212 to line 218 using else if (error.code ! = duplicatedUserNameCode), with this change we can remove the OR in the line 210.

**Comment by nicolasgranata** [comment]
Did we make any change in the signup.js file?

File: src/DonateTo.IdentityServer/Controllers/AccountController.cs [hunk 2]
```diff
@@ -203,6 +200,25 @@ private async Task<LoginViewModel> BuildLoginViewModelAsync(LoginInputModel mode
...
+            const string duplicatedUserNameCode = "DuplicateUserName";
+
+            foreach (var error in errors)
+            {
+                if (error.Code == duplicatedEmailCode)
+                {
+                    error.Description = "Email registered - please enter another email or sign in.";
+                    ModelState.TryAddModelError(error.Code, error.Description);
+                }
+                else if(error.Code != duplicatedUserNameCode)
```
Comment (nicolasgranata) [inline src/DonateTo.IdentityServer/Controllers/AccountController.cs:215]: add whitespace else if ()

File: src/DonateTo.IdentityServer/Views/Account/Register.cshtml [hunk 3]
```diff
@@ -1,44 +1,47 @@
...
-
-    <div class="form-group">
-        <input type="submit" value="Submit" class="btn btn-lg btn-primary btn-block submit-button" />
-    </div>
-</form>
-
-<script src="~/js/signup.js" asp-append-version="true"></script>
-@section Scripts {
-    @{await Html.RenderPartialAsync("_ValidationScriptsPartial");}
+﻿@model UserRegistrationViewModel
```
Comment (nicolasgranata) [inline src/DonateTo.IdentityServer/Views/Account/Register.cshtml:1]: Did we made any change in the signup.js file to display a popup or something like that?

Comment (Lordapa) [reply src/DonateTo.IdentityServer/Views/Account/Register.cshtml:1]: i add a summary with the display error

**Pull request merged**